custom-metrics-stackdriver-adapter/adapter.go

```diff
@@ -211,6 +215,10 @@ func main() {
 		"Stackdriver Endpoint used by adapter. Default is https://monitoring.googleapis.com/")
 	flags.BoolVar(&serverOptions.EnableDistributionSupport, "enable-distribution-support", serverOptions.EnableDistributionSupport,
 		"enables support for scaling based on distribution values")
+	flags.DurationVar(&serverOptions.ExternalMetricCacheTTL, "external-metric-cache-ttl", serverOptions.ExternalMetricCacheTTL,
+		"The duration (e.g., 1m, 5s) for which external metric values are cached.")
+	flags.IntVar(&serverOptions.ExternalMetricCacheSize, "external-metric-cache-size", serverOptions.ExternalMetricCacheSize,
```

Comment:
Could we set a default value here? I think 300 items here is more than enough.

This would effectively represent 300 HPAs in the cluster all using external metrics which is plenty, and folks who need it can boost this higher.

Reply:
Done